testing/jormungandr-integration-tests/src/common/jormungandr/mod.rs

```diff
@@ -12,19 +12,16 @@ use jormungandr_testing_utils::testing::MemPoolCheck;
 pub use process::*;
 pub use starter::*;
 use std::collections::HashMap;
-use std::path::PathBuf;
 use thiserror::Error;
 
 use jormungandr_testing_utils::testing::{FragmentNode, FragmentNodeError};
 
 #[derive(Error, Debug)]
 pub enum JormungandrError {
-    #[error("error in logs. Error lines: {error_lines}, logs location: {log_location}, full content:{logs}")]
-    ErrorInLogs {
-        logs: String,
-        log_location: PathBuf,
-        error_lines: String,
-    },
+    #[error("error in logs. Error lines: {error_lines}, full content:{logs}")]
```

Comment:
```suggestion
    #[error("error in logs. Error lines: {error_lines}, full content: {logs}")]
```
Not really sure it's useful to put the full JSON content in a single error printout. Maybe the test fixture handling this error can re-parse the `logs` field and print the lines out nicely?

Reply:
Maybe we could reuse log formatting from `tracing_subscriber::fmt` but I'm not sure if it can be used independently, need to check.